### DISCUSSION TITLE
Keep the players around for longer! Introduce two AC Modules that make Altoholism more appealing.

### DISCUSSION THREAD

**Discussion by Aethereus-ManaOnion** [chromiecraft/chromiecraft#5809] [Suggestions]
I feel very passionate about this server and our mission.

This discussion is a result of my grudge that I hold against the way WoW was designed pre-MoP, and my love for the QoL that they introduced in Pandaria.

I think having mounts, titles and achievements account bound / account wide would be an overwhelmingly positive change that our community desperately needs. Let me elaborate:

Currently, we are experiencing a problem with player retention and I believe it  largely has to do with Game Design Philosophy. In our version of the game making alts or rerolling feels like a waste of time, due to the fact that it feels like the time invested into playing the original toon goes completely to waste. All the achievements, titles that come as a result, mounts that were grinded to the death, all become a thing of a past. I am speaking from experience and I believe most players feel the same as me.

What we have now is a system that makes it stupid to have more than one character on a single account. It's way more incentivizing to have 4 burner accounts with separate characters due to the RaF system abuse and the ability to be logged into two clients at the same time. This is a subpar solution. From the player's perspective, I can say that when I think about making a new toon I always feel tempted to try my luck somewhere else and always just give up on the idea. The only reason I feel that incentivizes me now to run an alt is to have access to gathering professions, and it's not a strong enough incentive, for me and for many more others who feel like me.

That's why I want to put forward a proposition to include the two following modules from AzerothCore Repository that I believe would be a solution to many of those grudges and an overall improvement of the player experience.

mod-account-achievements (achievements, and titles by proxy): https://www.azerothcore.org/catalogue-details.html?id=348862524#/details/108285701

mod-accountbound (mounts, companions and heirlooms): https://www.azerothcore.org/catalogue-details.html?id=348862524#/details/465454676

I have previous experience of playing on servers with such modules on, and I can testify that the difference those made was absolutely incredible. I loved them ever since I first saw that they are possible to have. 

I have done extensive market research and opinion polling in our playerbase, both in World Chat and on Discord, and each time I brought this issue up the response was overwhelmingly positive.

The other argument is that both of those modules are ready-made and do not need overwhelming developer attention apart from some tuning probably.

Please consider my suggestion. I really believe it would be one of the best QoL changes in CC history and attract a lot of positive feedback and attention.

**Comment by konopkap** [comment]
Really good idea. Hope it'll get enough votes to consider by devs!

**Comment by LiliTheHuntress** [comment]
Totally agree, that would be very helpful! Having mounts, titles and achievements account bound / account wide would be indeed an overwhelmingly positive change.

**Comment by cachocarne** [comment]
I understand that it could cause some issues if one toon has half of some achievements and another has the other half thus that account obtaining "full achievements" as result. For example, think about opposite reputations and so on. What if you have mounts from Alliance on a toon and from Horde on the other? What about the achievement that involves killing leaders of the other faction? And fishing in SW/Org? Just to mention some examples that come to mind.

Nevertheless, I believe there could be a workout around it. One idea would be that achievements, mounts, pets, etc have to be farmed on ONE main character to be available for the alts. So if your main has it, they all have it. If your alt has it, your main or other alts won't. It would absolutely be a positive and joyful addition.

**Reply by Aethereus-ManaOnion** [reply]
Thank you for posting this opinion. It's much appreciated.

I don't think this module counts the partial completion of an achievement between characters. On Sunwell which was running this module, it meant that if I get Explorer of Durotar achievement on my Orc character, then my alt who is a Blood Elf has the achievement too. But if my blood elf got all locations in Ghostlands discovered apart from Tranquillen, then the achievement is not complete so my Orc is not entitled to the achievement. As for the cross-faction aspect you raise, I do not know if a solution for that is included, but I would expect that there are safeguards for that in the module. I would gladly test this on PTR if the mod would be temporarily enabled there for testing.

Another issue that comes to mind is, other classes getting paladin / warlock mounts too. I don't yet know the workaround for that, but I think those are minor issues that could probably easily be solved.

**Reply by konopkap** [reply]
From what I read fe suggested module's readme -> the faction specific mounts are replaced with equivalent ones.

About class mounts nothing is mentioned. I see in code that mount has "allowedClass" field - but don't know how it will behave (I guess you'll see this mount but you won't be able to summon it)

**Reply by Aethereus-ManaOnion** [reply]
@konopkap Thank you for delving into the details.

On the Sunwell server I played on which had this or similar module, this was still the case, that when I made a DK character, my paladin had access to the Acherus Deathcharger. I would blame that on sloppy development and unwillingness to correct and improve on their part (Sunwell was sort of a scam). 
I haven't tested the hypothesis yet, but as far as I understand the principle, mounts that would require a certain level of skill are gated behind particular mechanics in the game. If I'm not mistaken, for example engineering chopper, or tailoring flying carpet would still be gated to characters with sufficient profession skill to summon the mount. That would be ideal. On the other hand there is the case of mounts gated behind reputation, such as the Cenarion Expedition Hippogryph. If it would require a character to reach exalted to summon, then fair game. If not, I still think it would be a net benefit and a welcome addition, not forcing a player to grind for a mount they have already dedicated time to, making it more fun (TM). If that approach is contrary to our values, then there sure is a way to code it into the module to make exemptions in such cases, which I think would be a solution, and a good solution at that,

**Comment by hofik19** [comment]
I would love to check and play other classes but I'm held back by feeling that all my previous progress would be in some form wiped. Those two modules would be step in right direction, additionally I would add an option to share reputation account-wide

**Reply by Aethereus-ManaOnion** [reply]
I'm not entirely sold on the reputation side of things, because contrary to things like mounts and achievements, a lot of mechanics are gated behind reputation, so it would in essence have impact on how good your character is. If you have access to best gear and enchants immediately after you hit 70 it is impacting the gameplay, while having the Jenkins title at level 10 doesn't make the game easier in any way. Secondly as there is no ready made module for sharing reputation, it would require way more developer effort to introduce it.

Thank you for expressing your opinion on this topic. It's quite important to me.

**Reply by konopkap** [reply]
Yeah - reputation could be really tricky one to implement.

**Comment by Jbubby** [comment]
the more alt friendly things the better

**Comment by balleny** [comment]
I think for this to be considered in the first place, we would need it to be introduced to the PTR realm and then some dedicated testing and fixing. I could assume that the mods may need some bugfixing or adjustments to properly work with cross faction being enabled (as you should likely not be able to use some faction specific mounts with chars of the other faction)
